Content/StructureResolver.php

```diff
@@ -133,14 +156,19 @@ public function resolveProperties(
                     );
                 }
             } else {
-                if (!$structure->hasProperty($sourceProperty)) {
+                $propertyStructure = $structure;
+                if ('title' === $sourceProperty) {
```

Comment:
would use the `getProperties` proposed above here too, then we dont need to handle the `title` differently